conda/recipes/jupyterlab-nvdashboard/meta.yaml

```diff
@@ -20,35 +23,32 @@ build:
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python {{ requires_python }}
     - pip
     - nodejs
-    - hatch-jupyter-builder >=0.5
-    - hatch-nodejs-version >=0.3.2
-    - hatchling >=1.5.0
-    - jupyterlab >=4.0.0
+    {% for r in pyproject_toml["build-system"]["requires"] %}
+    - {{ r }}
+    {% endfor %}
```

Comment:
With the exception of some base requirements (`python`, `pip` & `nodejs`), we load all other build requirement from `pyproject.toml`